packages/core-v2/src/semantic-model/concepts/concepts-utils.ts

```diff
@@ -14,4 +19,52 @@ export function isSemanticModelRelationship(resource: Entity | null): resource i
 export const SEMANTIC_MODEL_GENERALIZATION = "generalization"; // todo use proper IRI
 export function isSemanticModelGeneralization(resource: Entity | null): resource is SemanticModelGeneralization {
     return resource?.type.includes(SEMANTIC_MODEL_GENERALIZATION) ?? false;
-}
\ No newline at end of file
+}
+
+export function isSemanticModelAttribute(resource: Entity | null): resource is SemanticModelRelationship {
+    if (!isSemanticModelRelationship(resource)) {
+        return false;
+    }
+    const [end1, end2] = resource.ends;
+
+    return (
+        (end1?.iri != null &&
+            end2?.iri == null &&
+            semanticModelRelationshipEndConceptIsNullEmptyOrDataType(end1, resource)) ||
+        (end2?.iri != null &&
+            end1?.iri == null &&
+            semanticModelRelationshipEndConceptIsNullEmptyOrDataType(end2, resource))
+    );
+}
+
+function semanticModelRelationshipEndConceptIsNullEmptyOrDataType(
+    resourceEnd: SemanticModelRelationshipEnd,
+    resource: SemanticModelRelationship
+) {
+    return (
+        resourceEnd.concept == null ||
+        resourceEnd.concept == "" /* TODO: dataType and other logic */ ||
+        resource.id.endsWith("#attribute")
```

Comment:
What is the reason for this?

Reply:
Temporary workaround for SKOS #449, don't have a better idea how to do it